homeassistant/components/airnow/sensor.py

```diff
@@ -4,9 +4,10 @@
 
 from collections.abc import Callable
 from dataclasses import dataclass
-from datetime import datetime
 from typing import Any
 
+from dateutil import parser
```

Comment:
```suggestion
from dataclasses import dataclass
from dateutil import parser
from typing import Any
```

nit: formatting

Reply:
Weird I thought precommit would order imports, so I didn't check it. Thanks for pointing it out.